src/components/controls/Stimulus.vue

```diff
@@ -0,0 +1,22 @@
+<template>
+  <v-container fluid>
+        <v-col :align='this.align'>
+            <v-row>
+                <v-label>{{label}}</v-label>
```

Comment:
I'm not sure whether this should be here at all, really but if it is, it should be optional, i.e. no extra row containing empty text